### PR TITLE
Soar integration updates

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds new sections to the following article:
[https://help.sumologic.com/docs/cloud-soar/automation/](https://help.sumologic.com/docs/cloud-soar/automation/)

The new sections are for new Cloud SOAR features:
- "Certified integrations"
- "Duplicate integrations"
- "Deleted bookmark"
- "Configure Slack for Cloud SOAR"

Content was provided by @AlexandruuPopa. 

Issue number: 
See this Asana ticket:
[https://app.asana.com/0/1183741177809740/1204292438065424/f](https://app.asana.com/0/1183741177809740/1204292438065424/f)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft.

**Comment by jpipkin1** [comment]
@AlexandruuPopa, could you take a look at this PR and let me know if it needs any changes?

File: docs/cloud-soar/automation.md [hunk 1]
```diff
@@ -454,9 +494,117 @@ Netwitness integration by uploading the new integration action file. This new ac
...
+   1. Paste your previously saved Bot User OAuth Access Token to the **Bot OAuth** field.
+   1. Paste your previously saved User OAuth Access Token to the **Oauth Token for channel creation** field. If configuration is successful, **Workspace** displays "Success".
+
+If your new resources are configured correctly, and Instant Messaging displays a "Success" message for the configured workspace, you can [Use the Slack app in User Choice](#use-the-slack-app-in-user-choice). 
+
+#### Use the Slack app in User Choice
+ 
+ If you have configured Slack as described in [Configure Slack for Cloud SOAR](#configure-slack-for-cloud-soar), you can set a playbook’s [User Choice](#user-choice) to be answered by Slack.
+
+1. Run a playbook with a User Choice action. The following example shows a simple playbook with two available answers: "Close Incident" and "Investigate". Notice that the option "Answer By Slack" is enabled.<br/><img src={useBaseUrl('img/cloud-soar/integration-slack-playbook.png')} alt="Playbook with user choices" width="600"/>
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:594]:
```suggestion
1. Run a playbook with a User Choice action. The following example shows a simple playbook with two available answers: **Close Incident** and **Investigate**. Notice that the option **Answer By Slack** is enabled.<br/><img src={useBaseUrl('img/cloud-soar/integration-slack-playbook.png')} alt="Playbook with user choices" width="600"/>
```

File: docs/cloud-soar/automation.md [hunk 2]
```diff
@@ -454,9 +494,117 @@ Netwitness integration by uploading the new integration action file. This new ac
...
+<br/>In this case, the Authorizer set is just a user. If a group is chosen, a Slack channel will be created and all the group members will be authorized to choose one of the User Choice available options. The channel will be automatically named as the incident on which the playbook is running. 
+1. When the playbook flow reaches the User Choice, the user or group will receive a message containing the reference to the incident, the playbook name, and the question set for the User Choice.<br/><img src={useBaseUrl('img/cloud-soar/integration-slack-user-choice.png')} alt="Slack user choice message" width="600"/>
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:596]:
```suggestion
<br/>In this case, the Authorizer set is just a user. If a group is chosen, a Slack channel will be created and all the group members will be authorized to choose one of the **User Choice** available options. The channel will be automatically named as the incident on which the playbook is running. 
1. When the playbook flow reaches the **User Choice**, the user or group will receive a message containing the reference to the incident, the playbook name, and the question set for the **User Choice**.<br/><img src={useBaseUrl('img/cloud-soar/integration-slack-user-choice.png')} alt="Slack user choice message" width="600"/>
```

**Review by kimsauce — APPROVED** [review]
Amazing job! Just two minor suggestions but looks great.

**Review by kimsauce — COMMENTED** [review]
Amazing job! Just two minor suggestions but looks great.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce.

**Review by AlexandruuPopa — APPROVED** [review]

**Pull request merged**